src/applications/facility-locator/components/search-results-items/Covid19Result.jsx

```diff
@@ -65,20 +64,19 @@ const Covid19Result = ({
           )}
         <LocationAddress location={location} />
         <LocationDirectionsLink location={location} from="SearchResult" />
-        {showCovidVaccineSchedulingLinks &&
-          covidSchedulingAvailable && (
-            <a
-              className="vads-c-action-link--blue vads-u-margin-bottom--1 vads-u-display--inline-block vads-u-margin-top--0"
-              href="/health-care/schedule-view-va-appointments/appointments/"
-              onClick={() =>
-                recordEvent({
-                  'cta-action-link-click': 'fl-schedule-covid-vaccine',
-                })
-              }
-            >
-              Schedule an appointment online
-            </a>
-          )}
+        {covidSchedulingAvailable && (
```

Comment:
@maxx1128 I just wanted to double check that the flipper `showCovidVaccineSchedulingLinks` is current "On" in Flipper?

I ask because I am not seeing anything related to covid vaccines on these pages, which makes me wonder if "on" is the current state of that flipper.
https://www.va.gov/find-locations/facility/vha_516GF
https://www.va.gov/find-locations/facility/vha_664GB
https://www.va.gov/find-locations/facility/vha_516BZ

Reply:
Also not seeing it here... which makes me wonder if the defualt state is off, which means we need to remove what is in the conditional.
![image](https://user-images.githubusercontent.com/5752113/203438046-9a024c31-b81e-4974-8532-d78bbaf0faf8.png)

Reply:
According to https://api.va.gov/flipper/features, it's turned off.

<img width="1067" alt="Screen Shot 2022-11-23 at 10 41 06 AM" src="https://user-images.githubusercontent.com/10790736/203588045-e8b17e5f-4f4b-402b-ab13-e9d9404eb352.png">

Reply:
@swirtSJW The conditional still checks `covidSchedulingAvailable`, which is based on different attributes from the facility's location data. This change would make the scheduling appear on the frontend based only on that, so we should probably keep the code in the conditional.

Reply:
Ok so if the flipper is  currently **off**, that code in the conditional is not firing.  So no-one is ever seeing the output.  It is off.  If we go with just removing that one element of the conditional, we are defaulting to **on**.  Which creates a change in the output of the FE.  Removing a toggle that has outlived its purpose and is no longer needed, should not result in a change to the FE experience.